src/devtools/client/webconsole/components/FilterBar/__tests__/FilterBar.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { render, createTestStore } from "test/testUtils";
+
+describe("First example test", () => {
+  it("Creates a store without exploding", async () => {
+    const store = await createTestStore();
+    expect(store.getState().consoleUI).toBeTruthy();
+  });
```

Comment:
Isn't it the idea to auto-wrap renders with context that is a store (+ provide partial initial state) so that we can just render without having to explicitly sync on store creation? Or because of other things that may need initializing, it's fine to sync manually on it?

Reply:
In an actual test, this will be:

```js
const { getByText } = render(<MyComponent />)
```

But for this first PR I just wanted to verify that creating the store runs without exploding, and loads the lazy reducers.

Reply:
Got it. Cool cool cool 👍